contracts/UniswapV3Staker.sol

```diff
@@ -6,13 +6,13 @@ import './interfaces/IUniswapV3Staker.sol';
 import './libraries/IncentiveId.sol';
 import './libraries/RewardMath.sol';
 import './libraries/NFTPositionInfo.sol';
+import './libraries/TransferHelperExtended.sol';
```

Comment:
we can remove the `import '@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol';` line now

Reply:
ahead of ya, unless there's a place I missed

Reply:
oops no sorry i think i was a commit behind or just missed that